src/main/java/ru/yanmayak/TelegramNotifyBot/bot/TelegramBot.java

```diff
@@ -0,0 +1,45 @@
+package ru.yanmayak.TelegramNotifyBot.bot;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+@Component
+public class TelegramBot extends TelegramLongPollingBot {
+
+    @Value("${bot.token}")
+    private String botToken;
+
+    @Value("${bot.username}")
+    private String botUsername;
+
+    @Override
+    public void onUpdateReceived(Update update) {
+        if (update.hasMessage() && update.getMessage().hasText()) {
+            String text = update.getMessage().getText();
+            String chatId = update.getMessage().getChatId().toString();
+
+            SendMessage message = new SendMessage(chatId, text);
+
+            try {
+                execute(message);
+            }
+            catch (TelegramApiException e) {
+                e.printStackTrace();
```

Comment:
Use logger, i.e by enabling Slf4j annotation and using log.error()/log.info() etc